Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -167,16 +170,18 @@ class WebViewController: UIViewController, WebController {
 
         browserView.configuration.userContentController.removeScriptMessageHandler(forName: "focusTrackingProtection")
         browserView.configuration.userContentController.removeScriptMessageHandler(forName: "focusTrackingProtectionPostLoad")
+        browserView.configuration.userContentController.removeScriptMessageHandler(forName: "findInPageHandler")
```

Comment:
nit: the existing code is at-fault with regards to my nit here, not your patch, however since there was a fatal crash in this code, it might be fair game to say we should consider getting this code as lean as possible. 
I think it is better to avoid repeated lines of logic (and the risk of typos in string constants),
so if the class had 

`let scriptHandlers = (tpPreload: "..."., tpPostLoad: "...", findInPage: "...")`
then one could do

`[scriptHandlers.tpPreload, scriptHandelers.tpPostLoad, scriptHanders.findInPage].forEach {
      browserView.configuration.userContentController.removeScriptMessageHandler(forName: $0)
 }`

Another code-repeat avoidance would be to have: 
`private func addScript(forResource: String, injectionTime:, forMainFrameOnly:) { ... }` 
replacing lines 151-153, 156-158, 162-165.